src/index.js

```diff
@@ -2,6 +2,7 @@ import './main.css'
 
 let isMaximizing = false
 let GOOGLE_ANALYTICS_DELAY_MS = 30
+let JOIN_STRIKE_LINK_URL = 'https://globalclimatestrike.net'
```

Comment:
let's call this GLOBAL_CLIMATE_STRIKE_LINK_URL -- we might make it about something other than joining a strike, so this more general name will still make sense.

Reply:
👍